apps/namadillo/src/App/Common/InactiveChannelWarning.tsx

```diff
@@ -0,0 +1,57 @@
+import { chainTokensAtom } from "atoms/chain/atoms";
+import {
+  getDenomFromIbcTrace,
+  ibcChannelsFamily,
+  searchChainByDenom,
+} from "atoms/integrations";
+import { useAtomValue } from "jotai";
+
+const Warning = ({
+  trace,
+  denom,
+}: {
+  trace: string;
+  denom: string;
+}): JSX.Element => {
+  const chainName = searchChainByDenom(denom)?.chain_name ?? "";
+  const { data: ibcChannels } = useAtomValue(ibcChannelsFamily(chainName));
+
+  // skip the warning if it's the current open channel
+  if (
+    !ibcChannels?.namadaChannel ||
+    trace.includes(ibcChannels.namadaChannel)
+  ) {
+    return <></>;
+  }
+
+  return <div className="text-xs text-neutral-500">inactive: {trace}</div>;
+};
+
+export const InactiveChannelWarning = ({
```

Comment:
Maybe we can extract the logic of this function and create a new one to return if a given token is active or not. Might be useful in other places